docs/src/SUMMARY.md

```diff
@@ -0,0 +1,19 @@
+# Summary
+
+[Foreword](foreword.md)
+[Getting Started](getting-started.md)
+
+- [Introduction](introduction/index.md)
+	- [Components](introduction/components.md)
+- [Ballot-Leader Election](ble/index.md)
+    - [Configuration](ble/configuration.md)
+    - [Tick Timer](ble/timer.md)
```

Comment:
Since these are so short, remove them and put everything in BLE under different headers instead.